dts/bindings/binding-template.yaml

```diff
@@ -56,15 +56,23 @@ sub-node:
 #
 #   <property name>:
 #     category: <required | optional>
-#     type: <string | int | boolean | array | uint8-array | string-array | compound>
+#     type: <string | int | boolean | array | uint8-array | string-array | phandle | compound>
 #     description: <description of the property>
 #
-# 'uint8-array' is our name for what the device tree specification calls
+# 'type: uint8-array' is for what the device tree specification calls
 # 'bytestring'. Properties of type 'uint8-array' should be set like this:
 #
 #   foo = [89 AB CD];
 #
 # Each value is a byte in hex.
+#
+# 'phandle' is for properties that are assigned a single phandle, like this:
```

Comment:
Please also regard multiple phandles as phandle-array for the pinctrl-x property